package/gluon-tunneldigger-watchdog/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+PIDFILE=/var/run/tunneldigger.mesh-vpn.pid
+
+if [ "$(uci get tunneldigger.@broker[0].enabled)" == "1" ]; then
```

Comment:
`==` is invalid in `[]` expressions (but accepted by many implementations, sometimes with a warning), this should be `=`.